bitnami/concourse/values.yaml

```diff
@@ -0,0 +1,1125 @@
+    ##
+    affinity: {}
+
+    ## Node labels for pod assignment
+    ## Ref: https://kubernetes.io/docs/user-guide/node-selection/
+    ##
+    nodeSelector: {}
+    ## @param web.tolerations Tolerations for web pods assignment
+    ## ref: https://kubernetes.io/docs/concepts/configuration/taint-and-toleration/
+    ##
+    tolerations: []
+    ## @param web.updateStrategy.type web statefulset strategy type
+    ## ref: https://kubernetes.io/docs/concepts/workloads/controllers/statefulset/#update-strategies
+    ##
+    updateStrategy:
+      ## StrategyType
+      ## Can be set to RollingUpdate or OnDelete
+      ##
+      type: RollingUpdate
+    ## @param web.priorityClassName web pods' priorityClassName
+    ##
+    priorityClassName: ""
+    ## @param web.lifecycleHooks for the web container(s) to automate configuration before or after startup
+    ##
+    lifecycleHooks: {}
+    ## @param web.extraEnvVars Array with extra environment variables to add to web nodes
+    ## e.g:
+    ## extraEnvVars:
+    ##   - name: FOO
+    ##     value: "bar"
+    ##
+    extraEnvVars: []
+    ## @param web.extraEnvVarsCM Name of existing ConfigMap containing extra env vars for web nodes
+    ##
+    extraEnvVarsCM:
+    ## @param web.extraEnvVarsSecret Name of existing Secret containing extra env vars for web nodes
+    ##
+    extraEnvVarsSecret:
+    ## @param web.extraVolumes Optionally specify extra list of additional volumes for the web pod(s)
+    ##
+    extraVolumes: []
+    ## @param web.extraVolumeMounts Optionally specify extra list of additional volumeMounts for the web container(s)
+    ##
+    extraVolumeMounts: []
+    ## @param web.sidecars Add additional sidecar containers to the web pod(s)
+    ## e.g:
+    ## sidecars:
+    ##   - name: your-image-name
+    ##     image: your-image
+    ##     imagePullPolicy: Always  
+    ##     ports:
+    ##       - name: portname
+    ##         containerPort: 1234
+    ##
+    sidecars: {}
+    ## @param web.initContainers Add additional init containers to the web pod(s)
+    ## ref: https://kubernetes.io/docs/concepts/workloads/pods/init-containers/
+    ## e.g:
+    ## initContainers:
+    ##  - name: your-image-name
+    ##    image: your-image
+    ##    imagePullPolicy: Always
+    ##    command: ['sh', '-c', 'echo "hello world"']
+    ##
+    initContainers: {}  
+  
+  baggageclaim:
+    ## Minimum level of logs to see. Possible values: debug, info, error
+    ##
+    logLevel: info
+
+    ## IP address on which to listen for API traffic.
+    ##
+    bindIp: 127.0.0.1
+
+    ## Port on which to listen for API traffic.
+    ##
+    bindPort: 7788
+
+    ## IP address on which to listen for the pprof debugger endpoints.
+    ##
+    debugBindIp: 127.0.0.1
+
+    ## Port on which to listen for baggageclaim pprof server.
+    ##
+    debugBindPort: 7787
+
+    ## Disable remapping of user/group IDs in unprivileged volumes.
+    ##
+    disableUserNamespaces:
+
```

Comment:
I ran an automated check and see that these values from the template are missing here (there may be false positives)

```
Key concourse.serviceAccount.name not found
Key concourse.web.baseResourceTypeDefaults not found
Key concourse.worker.baggageclaim.logLevel not found
Key externalDatabase.postgresqlUsername not found
Key persistence.worker.storageClass not found
Key podSecurityPolicy.allowedWebVolumes not found
Key postgresql.nameOverride not found
Key rbac.apiVersion not found
Key secrets.workerAdditionalCerts not found
Key web.baseResourceTypeDefaults not found
Key web.debugBindPort not found
Key web.enableAcrossStep not found
Key web.enableCacheStreamedVolumes not found
Key web.enablePipelineInstances not found
Key web.ingress.certManager not found
Key web.ingress.extraHosts not found
Key web.ingress.extraTls not found
Key web.ingress.hostname not found
Key web.ingress.path not found
Key web.ingress.pathType not found
Key web.tsa.bindDebugPort not found
Key worker.additionalCertificates not found
Key worker.affinity not found
Key worker.autoscaling.builtInMetrics not found
Key worker.autoscaling.customMetrics not found
Key worker.autoscaling.maxReplicas not found
Key worker.autoscaling.minReplicas not found
Key worker.command not found
Key worker.customLivenessProbe not found
Key worker.customReadinessProbe not found
Key worker.extraEnvVars not found
Key worker.extraEnvVarsCM not found
Key worker.extraEnvVarsSecret not found
Key worker.extraVolumeMounts not found
Key worker.extraVolumes not found
Key worker.hostAliases not found
Key worker.initContainers not found
Key worker.lifecycleHooks not found
Key worker.nodeSelector not found
Key worker.priorityClassName not found
Key worker.readinessProbe.enabled not found
Key worker.sidecars not found
Key worker.tolerations not found
Key worker.tsa.hosts not found
Key worker.updateStrategy not found
```